viewing_party/party.py

```diff
@@ -1,23 +1,183 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(movie_title, genre, rating):
+    if movie_title == None or genre == None or rating == None:
+        return None
+    movie = {"title":movie_title, "genre":genre, "rating":rating}
+    return movie

```

Comment:
We can directly return a dictionary literal without creating `movie` first. Typically, when you create a variable and then immediately return it without doing anything with the variable, you can just return the value without creating the variable. 

```suggestion
    return {
	        "title": title, 
	        "genre": genre, 
	        "rating": rating 
	        } 
```